Josephine_commit/calculate_zone_capacity.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
I believe this file was added to git unintentionally since it's a duplicate of the other one

Reply:
you can use `git rm` to remove it.